x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_view.tsx

```diff
@@ -62,11 +64,13 @@ export const EngineView: React.FC = () => {
             pageViewTelemetry={tabId}
             pageHeader={{
               pageTitle: tabId,
-              rightSideItems: [],
+              rightSideItems: [<EngineViewHeaderActions />],
             }}
             engineName={engineName}
             isLoading={isLoadingEngine}
-          />
+          >
+            <DeleteEngineModal />
```

Comment:
Could we render this outside of the `Switch` so it's available for all engine view routes? not sure what the implications of this would be. This route handler is just a place holder for all the routes that are not yet implemented.

because we got rid of the tabs each page will render its own template now.